cmd/woodpecker-autoscaler/flags.go

```diff
@@ -38,6 +41,12 @@ var flags = []cli.Flag{
 		Usage:   "maximum amount of agents",
 		EnvVars: []string{"WOODPECKER_MAX_AGENTS"},
 	},
+	&cli.StringFlag{
+		Name:    "min-age",
+		Value:   optionMinAgeDefault,
+		Usage:   "minimum age of agents before terminating",
```

Comment:
Don't understand the question. Because I don't want the agent to get destroyed immediately. I want the agent to stay alive for `60m` because when I work on something, recreating the agent 10 times is not efficient.

Reply:
Okay, but do we need users to be able to set this to 60 minutes, 3 hours or can we simply set it to some good common value.

Reply:
I see no reason why we shouldn't make it configurable, and yes, at work we have multiple (drone) autoscalers that create agents with different lifetimes for a reason.

Reply:
If I get you correctly your use-case is that your agents require different time to boot, install and connect as you are probably changing the cloud-init data to do different time consuming tasks with multiple autoscalers?

Reply:
We discuss a micro change that don't harm in any case for a week now... My use case it that when I start to work on something Ill most likely to for more than 5 minutes and create a lot of PRs, merges, commits for some time. The default behavior was that the agent gets terminated every single time when no pending workload exists. Creating 10 PRs within an hour of work results in 10x shutting down and recreating the agent. Every agent creation takes around 1-2 minutes, even with the default cloud-init. That's wasting a lot of resources and time. However, I think I'm going to maintain a fork for now to speed up my transition. As said, the autoscaler is the core component to be able to use Woodpecker at all.

Reply:
@xoxys Its fine for me to add it, but its pretty hard if you just propose a change without explaining how it works. Explaining it with a short example helps reviewing a lot and allows others like me to further improve the system in a whole. Sounds like you are using it in a really advanced way, but sad that the community will not learn and improve much from it.

Reply:
I'm just complaining a bit about the way to communicate and the (unnecessary) feedback loop in that particular case. But just my two cents :zipper_mouth_face: